--- .targetinfo.orig    2024-07-16 00:14:18.000000000 +0200
+++ .targetinfo 2024-08-26 20:28:25.972402072 +0200
@@ -36536,11 +36536,11 @@
 Target-Name: x86_64
 Target-Arch: x86_64
 Target-Arch-Packages: x86_64
 Target-Features: boot-part display ext4 fpu gpio pci pcie pcmcia rootfs-part rtc squashfs targz usb vdi vhdx virtio vmdk
 Target-Depends: 
-Target-Optimization: -Os -pipe
+Target-Optimization: -Os -pipe -march=btver2
 CPU-Type: 
 Linux-Version: 5.15.162
 Linux-Release: 1
 Linux-Kernel-Arch: x86
 Target-Description:
@@ -36561,10 +36561,17 @@
 
 
 
 @@
 
+Target-Profile: APU2
+Target-Profile-Name: PC Engines APU2
+Target-Profile-Packages: 6in4 bash rng-tools luci luci-mod-rpc luci-ssl ntpd htop beep amd64-microcode irqbalance fstrim libsensors lm-sensors usbutils wpad kmod-ath9k kmod-ath10k ath10k-firmware-qca988x kmod-igb kmod-gpio-button-hotplug kmod-gpio-nct5104d kmod-hwmon-core kmod-crypto-hw-ccp kmod-pcengines-apuv2 kmod-leds-gpio kmod-pcspkr kmod-sound-core kmod-sp5100-tco kmod-usb-core kmod-usb-ohci kmod-usb-serial kmod-usb-storage kmod-usb-storage-uas block-mount fdisk smartmontools hdparm kmod-usb2 kmod-usb3 -kmod-e1000e -kmod-e1000 -kmod-r8169
+Target-Profile-Description:
+		PC Engines APU2 Embedded Board
+
+@@
 
 Source-Makefile: target/linux/zynq/Makefile
 Target: zynq
 Target-Board: zynq
 Target-Name: Xilinx Zynq 7000 SoCs
